Jenkinsfile

```diff
@@ -0,0 +1,113 @@
+#!groovy
```

Comment:
Completely optional, but Declarative Pipeline is nicer than Scripted Pipeline IMHO (even if some of it ends up being nested-scripted anyway)